internal/static/js/app.js

```diff
@@ -123,7 +156,7 @@ const updatePlots = () => {
             plot.update(xrange, data, shapes);
         }
     });
-}
+}, PlotsDef.sendFrequency||1000)
```

Comment:
why is the `sendFrequency` setting used here? it wasn't before and I don't understand why it's needed now. Updates are pushed from the server, so I'd expect browser-side updates to be performed when data gets pushed.

Reply:
This sets a minimum time for DOM redrawing, maybe I should set it smaller, such as 100